beacon-chain/forkchoice/doubly-linked-tree/unrealized_justification.go

```diff
@@ -104,14 +104,12 @@ func (s *Store) pullTips(state state.BeaconState, node *Node, jc, fc *ethpb.Chec
 		return jc, fc
 	}
 
-	ab, uj, uf, err := precompute.UnrealizedCheckpoints(state)
+	_, uj, uf, err := precompute.UnrealizedCheckpoints(state)
```

Comment:
Let's just remove `activeBalance` from `UnrealizedCheckpoints`. It's cleaner

Reply:
nice one!